lametro/models/cms.py

```diff
@@ -122,6 +127,9 @@ def content(self):
             + "</div>"
         )
 
+    class Meta:
+        ordering = ["pk"]
+
```

Comment:
This way they're always listed in chronological order of when they were added whether when rendered or when listed in the cms.